geode-benchmarks/src/main/java/org/apache/geode/benchmark/tasks/CreatePartitionedRegion.java

```diff
@@ -27,7 +33,14 @@
 
   @Override
   public void run(TestContext context) throws Exception {
-    Cache cache = (Cache) context.getAttribute("SERVER_CACHE");
-    cache.createRegionFactory(RegionShortcut.PARTITION_REDUNDANT).create("region");
+    final Cache cache = (Cache) context.getAttribute(SERVER_CACHE);
+    final PartitionAttributes<?, ?> partitionAttributes = new PartitionAttributesFactory<>()
+        .setTotalNumBuckets(1 << 8)
+        .setRedundantCopies(1)
+        .create();
+    final Region<?, ?> region = cache.createRegionFactory(RegionShortcut.PARTITION_REDUNDANT)
+        .setPartitionAttributes(partitionAttributes)
+        .create("region");
+    PartitionRegionHelper.assignBucketsToPartitions(region);
```

Comment:
Calling `assignBucketsToPartitions` in this task seems like it's going to potentially screw up the bucket balance. If each JVM calls CreatePartitionRegion in parallel, it's possible one JVM will finish first and get all of the buckets.

`assignBucketsToPartitions` should probably be a separate step.

Reply:
Actually, this is a good find. None of this change should be in this PR. This was an attempt at a like for like comparison between Geode APIs and Redis APIs. I will just be pulling this change out completely.